scripts/exregional_run_met_genensprod_or_ensemblestat.sh

```diff
@@ -180,16 +183,22 @@ if [ "${grid_or_point}" = "grid" ]; then
     case "${FIELDNAME_IN_MET_FILEDIR_NAMES}" in
       "APCP01h")
         OBS_INPUT_FN_TEMPLATE="${OBS_CCPA_APCP01h_FN_TEMPLATE}"
+        FCST_INPUT_DIR="${vx_fcst_input_basedir}"
+        ;;
+      "ASNOW")
+        OBS_INPUT_FN_TEMPLATE="${OBS_NOHRSC_ASNOW_FN_TEMPLATE}"
```

Comment:
@willmayfield I will try to run this PR myself to understand better what's going on, but can you say why `FCST_INPUT_DIR` for `ASNOW` is set to `vx_output_basedir`?  Is there a vx pre-processing task (like `PCP_COMBINE` for `APCPgt01h`) that places output in `vx_output_basedir`?

Reply:
@gsketefian Yes, pcpcombine_fcst runs for ASNOW prior to this task.

Reply:
Oh, just saw below that the answer to my question is "yes"!

Reply:
So what do the `PCP_COMBINE` tasks do exactly for `ASNOW`?  Are the hours in the obs every 6 hours, and those get combined to 12, (not 18?), 24, (not 30?), 36, etc?  And same for forecasts?

Reply:
The obs are available as 6- and 24-hour accumulations in grib2 format from nohrsc on HPSS (and those are the accumulations recommended to be verified in the [Metrics Spreadsheet](https://docs.google.com/spreadsheets/d/1WzA2Um-kfbGs0Bh9fvFRcxzBirKyu9AQ-aHKJwAItIc/edit?usp=sharing)). Therefore those are simply retrieved and pcp_bombine_obs is not needed. 

The forecasts, on the other hand, are hourly 1-h accumulations output in post. So they are run with pcp_combine_fcst to combine them into the 6- and 24-h accumulations to match the obs.